spec/lib/appsignal/logger_spec.rb

```diff
@@ -67,6 +67,18 @@
     end
   end
 
+  describe "#silence" do
+    it "calls the given block" do
+      num = 1
+
+      logger.silence do
+        num += 1
+      end
+
+      expect(num).to eq(2)
```

Comment:
```suggestion
      expect(num).to eq(2)
      expect(Appsignal::Extension).to_not receive(:log)
```

Could help to make sure we don't log anything through this method.